dashboard/app/entities.go

```diff
@@ -569,7 +569,8 @@ type Job struct {
 	Error       int64 // reference to Error text entity, if set job failed
 	Flags       dashapi.JobDoneFlags
 
-	Reported bool // have we reported result back to user?
+	Reported  bool // have we reported result back to user?
+	IsInvalid bool // has a user flagged the result as nonsense?
```

Comment:
What about `InvalidatedBy`/`InvalidatedLink` instead of `IsInvalid`? So that we could later point to the place where the job result was invalidated.

Reply:
InvalidatedBy sounds like a good idea. What do you mean with InvalidatedLink? What would that point to?

Reply:
To the email with the command (see `Link` in `reporting_email.go`).

Reply:
Currently there is no email though, it's a button on the dashboard. If we add an equivalent email command, we can add the link at that point. Agreed?

Reply:
sgtm